eng/Directory.Build.Common.props

```diff
@@ -116,14 +116,14 @@
     <SupportsNetStandard20 Condition="'$(SupportsNetStandard20)' == '' and '$(IsClientLibrary)' == 'true'">true</SupportsNetStandard20>
     <SupportsNetStandard20 Condition="'$(SupportsNetStandard20)' == ''">false</SupportsNetStandard20>
 
-    <RequiredTargetFrameworks>net452;netstandard1.4;net461;netstandard2.0</RequiredTargetFrameworks>
-    <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">netstandard2.0</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks>net7.0;net452;netstandard1.4;net461;netstandard2.0</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">net7.0;netstandard2.0</RequiredTargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true' or '$(IsSamplesProject)' == 'true' or '$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
     <IsPackable>false</IsPackable>
     <!-- List newest targets first so that recordings are made with latest, running tests from editor runs latest, etc. -->
-    <RequiredTargetFrameworks>net6.0;netcoreapp3.1</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks>net7.0;net6.0;netcoreapp3.1</RequiredTargetFrameworks>
```

Comment:
@heaths:  Is this supplanting your PR for updating the SDK and targets?   Seems we have some overlap.

Reply:
Yes, please don't do this yet. There's more work involved and I should be done this week. There's also a horrible regression in 7.0.100 that will cause all pipelines to run _way_ more than they need to (VSTest is broken; fixed in 7.0.101).